docs/src/content/docs/guides/recurring-subscription-incoming-amount.mdx

```diff
@@ -0,0 +1,550 @@
+  "completed": false,
+  "createdAt": "2025-10-14T00:00:50.52Z",
+  "methods": [
+    {
+      "type": "ilp",
+      "ilpAddress": "...",
+      "sharedSecret": "..."
+    }
+  ]
+}
+```
+
+</details>
+
+### 4. Request a quote grant
+
+Use the customer's `authServer` details, received in Step 1, to call the <Badge text="POST" variant="success" /> [Grant Request API](/apis/auth-server/operations/post-request).
+
+This call obtains an access token that allows you to request that a quote resource be created on the customer's wallet account.
+
+<Tabs syncKey="lang">
+<TabItem label='TypeScript/JavaScript' icon='seti:javascript'>
+  ```ts wrap
+  const customerQuoteGrant = await client.grant.request(
+    {
+      url: customerWalletAddress.authServer
+    },
+    {
+      access_token: {
+        access: [
+          {
+            type: 'quote',
+            actions: ['create']
+          }
+        ]
+      }
+    }
+  )
+  ```
+</TabItem>
+</Tabs>
+
+<details>
+<summary>Example response</summary>
+The following shows an example response from the customer's wallet provider.
+
+```json wrap
+{
+  "access_token": {
+    "value": "...", // access token value for quote grant
+    "manage": "https://auth.cloudninebank.example.com/token/{...}", // management uri for access token
+    "access": [
+      {
+        "type": "quote",
+        "actions": ["create"]
+      }
+    ]
+  },
+  "continue": {
+    "access_token": {
+      "value": "..." // access token for continuing the request
+    },
+    "uri": "https://auth.cloudninebank.example.com/continue/{...}" // continuation request uri
+  }
+}
+```
+
+</details>
+
+### 5. Request the creation of a quote resource
+
+Use the access token received in the previous step to call the <Badge text="POST" variant="success" /> [Create Quote API](/apis/resource-server/operations/create-quote).
+
+This call requests that a quote resource be created on the customer's wallet account. The request must contain the `receiver`, which is the service provider's incoming payment `id`, along with the `incomingAmount`, which is the exact amount the service provider wants to receive.
+
+The `incomingAmount` specifies that the service provider will receive exactly \$15 USD.
+
+<Tabs syncKey="lang">
+<TabItem label='TypeScript/JavaScript' icon='seti:javascript'>
+  ```ts wrap
+  const customerQuote = await client.quote.create(
+    {
+      url: customerWalletAddress.resourceServer,
+      accessToken: customerQuoteGrant.access_token.value
+    },
+    {
+      method: 'ilp',
+      walletAddress: customerWalletAddress.id,
+      receiver: serviceProviderIncomingPayment.id,
+      incomingAmount: {
+        value: '1500',
+        assetCode: 'USD',
+        assetScale: 2
+      }
+    }
+  )
+```
+</TabItem>
+</Tabs>
+
+The response returns a `receiveAmount`, a `debitAmount`, and other required information.
+
+- `debitAmount` - The amount the customer must pay.
+- `receiveAmount` - The amount the service provider will receive.
```

Comment:
In the OTP guide for this property, the description is "The `incomingAmount` value from the incoming payment resource." I know that's not applicable here, because then the value would be `18000`. Can you check if the description you have is OK, or if it should say something like "The `incomingAmount` value from the quote request."